### PR TITLE
PIL-1008 Remove Intercom integration & PIL-1007 Finish Instabug Implementation

### PR DESCRIPTION
**Pull request by divyaBacancy**

### CODE REVIEW

**Comment by divyaBacancy** [comment]
https://user-images.githubusercontent.com/66996635/118152115-f9aa0b00-b431-11eb-9626-b7b620eccfe3.mov

**Comment by divyaBacancy** [comment]
https://user-images.githubusercontent.com/66996635/118152508-6ae9be00-b432-11eb-9540-6336d7aa78c7.mp4

File: src/actions/authActions.js [hunk 1]
```diff
@@ -171,6 +165,11 @@ export const loginAction = (
       const keychainLogin = await canLoginWithPkFromPin(!!biometricsSetting);
       if (pin && keychainLogin) {
         wallet = await getWalletFromPkByPin(pin);
+        // eslint-disable-next-line i18next/no-literal-string
+        Instabug.setUserAttribute('address', wallet.address);
+        if (wallet.provider._network.ensAddress) {
+          Instabug.setUserAttribute('ENS', wallet.provider._network.ensAddress);
+        }
```
Comment (IAmKio) [inline src/actions/authActions.js:170]: @poocart do you know if `wallet.provider._network.ensAddress` is where the ENS address stored?

File: src/components/RecoveryPortalWebView/RecoveryPortalWebView.js [hunk 2]
```diff
@@ -50,10 +51,13 @@ const RecoveryPortalWebView = ({
   onRef,
   urlPath = '',
   goBackDismiss,
+  isKnowledgeBaseUrl,
 }: Props) => {
   let webViewRef;
   let canWebViewNavigateBack = false;
-  const uri = `${getEnv().RECOVERY_PORTAL_URL}/${urlPath}`;
+  // eslint-disable-next-line i18next/no-literal-string
+  const knowledgeBaseUrl = 'https://help.pillarproject.io/en/';
```
Comment (IAmKio) [inline src/components/RecoveryPortalWebView/RecoveryPortalWebView.js:59]: Could we add this `knowledgeBaseUrl` as a Firebase Remote Config value please? Please use the Firebase Remote Config parameter key `knowlegebase_url`.

File: src/screens/Services/Services.js [hunk 3]
```diff
@@ -377,7 +377,7 @@ class ServicesScreen extends React.Component<Props> {
       <ContainerWithHeader
         headerProps={{
           noBack: true,
-          rightItems: [{ link: t('button.support'), onPress: () => Intercom.displayMessenger() }],
+          rightItems: [{ link: t('button.support'), onPress: () => { Instabug.show(); } }],
```
Comment (IAmKio) [inline src/screens/Services/Services.js:380]:
```suggestion
          rightItems: [{ link: t('button.support'), onPress: () => Instabug.show() }],
```

Is this better?

**Review by IAmKio — CHANGES_REQUESTED** [review]
During the review i noticed that we removed a few language strings, e.g. `src/locales/en/auth.json `. These were used in the onboarding video below: 
https://user-images.githubusercontent.com/1009271/118156301-1b46ca80-b411-11eb-9c3a-e6c24f26e0c1.mp4

We're now missing this information when users onboard and we need to replace it with Instabug's version.

Instabug's Privacy Policy: https://instabug.com/privacy

**Comment by divyaBacancy** [comment]
@IAmKio Thank you for reviewing. The changes have been done.

File: src/actions/authActions.js (hunk 1 above)
Comment (IAmKio) [reply src/actions/authActions.js:170]: We'll revisit this when @poocart is less busy

**Review by IAmKio — APPROVED** [review]

File: src/actions/appActions.js [hunk 4]
```diff
@@ -268,6 +269,10 @@ export const setupSentryAction = (user: ?Object, wallet: Object) => {
         ethAddress: address,
       },
     });
+    // eslint-disable-next-line i18next/no-literal-string
+    Instabug.setUserAttribute('address', wallet.address);
```
Comment (IAmKio) [inline src/actions/appActions.js:273]:
```suggestion
    Instabug.setUserAttribute('address', address);
```

We already pulled out `address` above - we can just use that instead

File: src/actions/appActions.js [hunk 5]
```diff
@@ -268,6 +269,10 @@ export const setupSentryAction = (user: ?Object, wallet: Object) => {
...
+    if (wallet.provider._network.ensAddress) {
+      Instabug.setUserAttribute('ENS', wallet.provider._network.ensAddress);
+    }
```
Comment (IAmKio) [inline src/actions/appActions.js:276]:
```suggestion
    if (username) {
      Instabug.setUserAttribute('ENS', username);
    }
```

Suggesting as we have already destructured `username` from the incoming `user` parameter.

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just some minor changes here but other than that - we'll be good to go.

File: src/actions/appActions.js (hunk 5 above)
Comment (divyaBacancy) [reply src/actions/appActions.js:276]: Sure will do that.

**Pull request merged**